src/is-authenticated-with-context.js

```diff
@@ -0,0 +1,10 @@
+import Cookies from 'js-cookie'
+
+export default function () {
```

Comment:
Maybe i'm not understanding the use cases. I was imagining the following scenarios:

1. you want to do something IF the user is authenticated to the app, regardless of context -> `isAuthenticated()`

2. you want to do something ONLY IF the user is authenticated with a specific context -> `isAuthenticatedWithContext(myContext)`

3. you want to do something CONDITIONALLY based on the authenticated context -> `getAuthenticationContext()`

The simplest case is that the context is a string, however, you have it implemented as an object, and perhaps thats why you have the last 2 flipped, because you need flexibility to get at the value you want.

First, lets decide if we need an object for this or if a string will suffice, then the cookie would just be
```
{ token: <string>, context: <string> }
```

If we decide that we need an object, then the above pattern can still work by using a string path as the key (with help from Lodash) Ex.
```
const cookie = {
  token: 'coldcutcombo',
  context: {
    thursday: 'meatball',
    friday: {
      lent: 'fishy',
      other: 'italian'
    }
  }
}

isAuthenticatedWithContext('friday.other')
// 'italian'
```
where
```
import { get } from 'lodash/fp'
function isAuthenticatedWithContext(ctxPath) {
  return get(ctxPath, cookie)
}
```

Does this make sense? Do you have use cases that I'm not considering?